test/test_other.py

```diff
@@ -4771,18 +4779,13 @@ def test(cxx, no_exit, assertions, flush=0, keepalive=0, filesystem=1):
           for flush in [0, 1]:
             test(cxx, no_exit, assertions, flush)
 
-  def test_fs_after_main(self):
-    for args in [[], ['-O1']]:
-      print(args)
-      self.run_process([EMXX, test_file('fs_after_main.cpp')])
-      self.assertContained('Test passed.', self.run_js('a.out.js'))
-
-  def test_opt_levels(self):
-    for opt in ['-O1', '-O2', '-Os', '-Oz', '-O3', '-Og', '-Ofast']:
-      print(opt)
-      proc = self.run_process([EMCC, '-v', test_file('hello_world.c'), opt], stderr=PIPE)
-      self.assertContained(opt, proc.stderr)
-      self.assertContained('hello, world!', self.run_js('a.out.js'))
```

Comment:
Looks like this test was removed entirely? We may lose coverage for `-Ofast` due to that (`-Og` appears in another test outside of this diff).

Reply:
Yeah, I figured we have lots of testing for opt levels elsewhere already, but perhaps you are right that `-Ofast` and `-Og` are not well tested otherwise.